website/tests/acceptance/about-test.js

```diff
@@ -0,0 +1,22 @@
+import { module, test } from 'qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'website/tests/helpers';
+import { a11yAudit } from 'ember-a11y-testing/test-support';
+
+module('Acceptance | about', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('visiting /about', async function (assert) {
```

Comment:
_in theory_ we could imagine to get the list of pages at build time (see https://github.com/hashicorp/design-system/blob/main/website/lib/markdown/index.js#L65-L78) and loop over the list, in a single test file, instead of duplicating all the files, one for each page

/cc @Dhaulagiri @alex-ju

Reply:
Created https://hashicorp.atlassian.net/browse/HDS-1458 to track this 👍

Reply:
We did discuss this as an idea but decided the manual approach was more pragmatic for the time being